scientific/E3SM_Native_Grid_Test/E3SM_ne30_Native_Grid_Test.ipynb

```diff
@@ -0,0 +1,311 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## To Interact with the 3D image...\n",
+    "\n",
+    "You will ultimately need to run the Python code in this notebook in a command line window, not within this notebook. One possible work flow is:\n",
+    "\n",
+    "1. Step through this notebook line by line to make sure everything works (you are running an environment with the needed packages, the code is working for you with the sample data, etc.).\n",
+    "2. Once everything is working, either use the Python file we've provided above or save this notebook as a Python file by clicking on File (within the JupyterLab interface) > Export Notebook As... > Export Notebook to Executable Script.\n",
+    "3. If you save/export the script from this notebook, you will need to edit the .py file to remove the \"#\" symbol infront of the x.interact() command near the bottom of the file. (If you download the python script directly, no editing is necessary.)\n",
+    "4. Open a new command line window and activate your favorite CDAT environment (for testing this notebook, I used a CDAT 8.1 mesalib environment, but you could also use a [jupyter-vcdat environment after installing VCDAT](https://github.com/CDAT/jupyter-vcdat/wiki/Deploy). **(Note: When you run the Python script to create the interactive window of the 3D data, make sure you are using a \"regular\" version of the conda environment - NOT a mesalib version - since you will need to see the window that opens as the Python script runs. (Mesalib environments suppress this window.))**\n",
+    "5. Navigate to where the Python script is stored and type the following at the prompt:\n",
+    "```\n",
+    "python -i E3SM_ne30_Native_Grid_Test.py\n",
+    "```\n",
+    "The generic interactive Python command is:\n",
+    "```\n",
+    "python -i Name_of_Your_Notebook.py\n",
+    "```\n",
+    "\n",
+    "A separate window should open where you can adjust the sliders and click on the \"Configure\" button to access more features of the plot. Click and drag on the center of the plot to change the angle at which you are viewing the data.\n",
+    "\n",
+    "To exit the interactive mode, close the window and type control-D within your command line window fully exit the interactive mode and to get your command prompt back. (If for some reason control-D does not work, try control-Z.)\n",
```

Comment:
To exit the interactive mode, close the window and type control-D within your command line window to fully exit the interactive mode and to get your command prompt back.

Reply:
Thanks - fixed.